setup.py

```diff
@@ -54,11 +98,49 @@ def extension_common_args(*args, **kwargs):
 ]
 
 if enable_cuda:
-    ext_modules.append(extension_common_args("primitiv.devices._cuda_device",
-                                             sources=["primitiv/devices/_cuda_device.pyx"]))
+    ext_modules.append(
+        extension_common_args(
+            "primitiv.devices._cuda_device",
+            libraries=[
+                "cudart",
```

Comment:
Order library names by alphabetical order if there is no special reason.

Reply:
Fixed